src/manager/agents/common-module/module/utils/common_utils.cpp

```diff
@@ -121,37 +166,45 @@ bool CommonUtils::TransformToScStruct(
   return true;
 }
 
-ScAddrVector CommonUtils::GetNodesUnderParameter(
+CommonUtils::ScAddrUnorderedSet CommonUtils::GetComponentsToInstall(
```

Comment:
```suggestion
ScAddrUnorderedSet CommonUtils::GetComponentsToInstall(
```
You are already in common utils. Remove everywhere. Use using namespace if it is required